requirements-selftests.txt

```diff
@@ -12,6 +12,9 @@ funcsigs>=0.4
 aexpect==1.5.1
 psutil==5.4.7
 
+# To test optional_plugins/varianter_yaml_to_mux
+pyaml==19.12.0
```

Comment:
I was testing your patch and noticed that, maybe, it would also be a good idea to change `optional_plugins/varianter_yaml_to_mux/setup.py` with this same package added here.

Or we just add this PR and submit a new one later.

Reply:
Hummm nice catch. I will remove this yaml from this PR and open a new issue for a better investigation on pyaml/PyYaml problem.